src/modules/launcher/Wox.Infrastructure/Image/ImageCache.cs

```diff
@@ -34,7 +34,7 @@ public ImageCache()
             get
             {
                 Usage.AddOrUpdate(path, 1, (k, v) => v + 1);
-                var i = _data[path];
+                _data.TryGetValue(path, out ImageSource i);
```

Comment:
How do the ImageCache users handle receiving a null as an image source? (I'm assuming that's what this will do in case of failure, right?)
Won't receiving a null still lead to a cache, or will it just show a clear image?

Reply:
When _data[path] was empty, it would crash with KeyNotFoundException. Now it can continue to work by assigning a null value and showing an empty image.

Reply:
Can we log something when this fails, warn level? I suspect this wasn't supposed to be called without expecting, so I think we should log it in case issues appear where people complain no images are shown and we can see those log messages. Thank you.